Introduction/README.md

```diff
@@ -21,16 +21,68 @@ machine as Fn provides the necessary Go compiler and tools as a Docker
 container.  Let's walk through your first function to become familiar with the
 process and how Fn supports development.
 
-Before we start developing we need to set the `FN_REGISTRY`
-environment variable.  Normally, it's set to your Docker Hub username.
-However in this tutorial we'll work in local development mode so we can set
-the `FN_REGISTRY` variable to an arbitrary value. Let's use `fndemouser`.
+### Configure your Context
+Before we start developing we need to configure Fn to use a Docker registry.
+Normally, it's set to your Docker Hub username. However in this tutorial we'll
```

Comment:
This doesn't read quite right to me. Maybe something like "Before we start developing we need to configure Fn to point to an appropriate Docker registry so it knows where to push your function images to. Normally this should be configured to point to your Docker Hub account by specifying your Docker Hub username, but for pure local development, such as in this tutorial, we can simply configure with an arbitrary value such as 'fndemouser'."?

Obviously reword and correct as you see fit...